src/main/java/de/tum/in/www1/artemis/domain/iris/settings/IrisSubSettings.java

```diff
@@ -32,6 +32,10 @@ public class IrisSubSettings extends DomainObject {
     @Column(name = "preferredModel")
     private String preferredModel;
 
+    @Nullable
+    @Column(name = "rateLimit")
+    private Integer rateLimit;
+
```

Comment:
Yes it should be since the HestiaSubSettings do not have a rate limit. Also for now only Global Settings can have a rate limit.
This will be improved in a future PR that generally improves and refactors the Iris settings, as they are still very rough.
This will then include things like:
- Subclasses for the different setting types (global, etc)
- Subclasses for the different sub-setting types (chat, hestia)
- Properly coded defaults
- Better UI